test/util/bspNet/waits.ts

```diff
@@ -112,10 +112,19 @@ export const waitForBspVolunteerWithoutSealing = async (
  *
  * @throws Will throw an error if the expected extrinsic or event is not found.
  */
-export const waitForBspStored = async (api: ApiPromise, checkQuantity?: number) => {
+export const waitForBspStored = async (api: ApiPromise, checkQuantity?: number, bspId?: string) => {
+  // TODO: add bsp ID to check if is not confirming storing
```

Comment:
This was left dangling right?

Reply:
And actually, it should be the BSP Account ID (the address)

Reply:
I changed it for `bspAccount`